gadget/gadget_test.go

```diff
@@ -3614,10 +3615,13 @@ func (s *gadgetYamlTestSuite) TestGadgetInfoHasSameYamlAndJsonTags(c *C) {
 		return s.PkgPath == ""
 	}
 
-	tagsValid := func(c *C, i interface{}) {
+	tagsValid := func(c *C, i interface{}, skip []string) {
 		st := reflect.TypeOf(i).Elem()
 		num := st.NumField()
 		for i := 0; i < num; i++ {
+			if strutil.ListContains(skip, st.Field(i).Name) {
+				continue
+			}
```

Comment:
it seems we shouldn't skip the whole check but  reorg things and have a list that checks that tagYaml is "-" and tagJSON is not? otherwise if not in the list do the comparison check?